etc/build-crypto-pairings.bash.paul

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
@echupriyanov Do we need this file?  If so, please rename it to something to reflect its usage with an explanation, otherwise please delete.